cypress/e2e/models/migration/task-manager/task-manager.ts

```diff
@@ -42,12 +42,12 @@ export class TaskManager {
         selectItemsPerPage(itemsPerPage);
     }
 
-    static verifyStatus(application: string, status: string): void {
+    static verifyTaskStatus(application: string, kind: string, status: string): void {
```

Comment:
I think `kind` should be of  `TaskKinds` type here and `status` of type `TaskStatuses`